File: BECore/app/model/Rank.js
/*
* Danh mục cấp bậc
* Create by: dvthang:09.01.2018
*/
Ext.define('QLDT.model.Rank', {
    extend: 'Ext.data.Model',
    idProperty: 'RankID',
    idType: 'Guid',

    code: 'RankCode',
    name: 'RankName',
    fields: [
		{ name: "RankCode", type: "string" },
		{ name: "RankName", type: "string" },
		{ name: "Description", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

File: MT.Model/Dictionary/Rank.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MT.Model
{
    /// <summary>
    /// Bảng cấp bậc
    /// </summary>
    /// Create by: dvthang:08.01.2018
    public class Rank : EntityBase
    {
        [Key]
        public Guid RankID { get; set; }
        public string RankCode { get; set; }
        public string RankName { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
    }
}
